src/main.js

```diff
@@ -1 +1,173 @@
-alert("Testing script")
\ No newline at end of file
+//Using https://exchangerate.host/#/docs
+
+const $btnConsult = document.querySelector("#btn-consult");
+
+const $newConsultbtn = document.querySelector("#reset");
+
+handleSymbols();
+
+$btnConsult.onclick = validateForm;
+
+function validateForm(e) {
+  deletePreviousRates();
+  const $form = document.querySelector("#form");
+  const $title = document.querySelector("#title");
+  const $date = document.querySelector("#date").value;
+
+  if (isValidDate($date)) {
+    let date = document.querySelector("#date");
+    date.classList.remove("alert");
+    date.classList.remove("alert-danger");
+    let userBase = $form.currency.value;
+
+    getExchangeByBaseAndDate(userBase, $date);
+    const $exchangeTitle = document.querySelector("#rates-container h2");
+    $exchangeTitle.textContent = `Tipo de cambio con base ${userBase}:`;
+  }
+
+  if (!isValidDate($date)) {
+    let date = document.querySelector("#date");
+    let dateInfo = document.querySelector("#date-info");
+
+    date.classList.add("alert");
+    date.classList.add("alert-danger");
+    dateInfo.textContent = "La fecha ingresada no es válida";
+  }
+
+  e.preventDefault();
```

Comment:
this goes on top, if something fails before reaching this line, the form will submit regardless.